deps.edn

```diff
@@ -22,7 +22,8 @@
                                                            commons-io/commons-io
                                                            slingshot/slingshot]}
   clojurewerkz/quartzite                    {:mvn/version "2.1.0"               ; scheduling library
```

Comment:
Could we perhaps consider this instead? Since it would fix emails about old log4j too.
https://mvnrepository.com/artifact/gov.nasa.earthdata/quartzite
Something makes me think that updating quartz will likely also fix 10088, which has been painfully difficult to reproduce.

Reply:
Cool. Didn't realize NASA was using Clojure or had their own Quartzite fork.

There's no problem with Quartzite itself -- the Quartz API is super stable and Quartzite is just a Clojure wrapper around it. It would be **nice** if Quartzite was updated to point to the latest Quartz version, but it's easy enough to explicitly request it ourselves. It's actually preferrable to make it explicit anyway because when the next version comes out we won't have to wait for the upstream library to get updated. 

So for that reason I think we should just stick to what I did here. Maybe Quartz `2.3.3` comes out tomorrow. We'd be dependent on NASA bumping their fork of the lib, and we'd be stuck in the same boat as we are now. Better to have control over this stuff ourselves

Reply:
The NASA fork lives here if anyone else wants to look at it: https://github.com/daniel-zamora/quartzite/blob/master/project.clj

AFAIK the only difference is bumping the `quartz` version